### PR TITLE
Add `GB Cup 2023 Autumn` wiki articles

### PR DESCRIPTION
**Pull request by HanHan3z**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by yobrevelc — APPROVED** [review]

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+7. Qualifier stage's result will using to seeding and choosen captains.
+   - Higher scores lead to a higher chance to be picked during drafting phase. Players are eliminated if they do not get picked, or if they get too low of a score during Qualifers.
+8. Some best-performance players will advance to drafting phase.
+   - Disrupting the match by foul play or seed manipulation will result in disqualification, and any such players will be announced before the drafting phase takes place.
+9. Players will use same multi rooms if they select same time to play qualifier. It is prohibited to abort a match with the reason of technical problem. With this situation, the score will be recorded normally.
+
+### Stage instructions
+
+1. Following the Qualifier stage, all player should participate a phase to form teams , called *Drafting phase*. Both zones will have their independent drafting. All the unpicked player will be eliminated from the tournament.
+2. Following the drafting phase, a swiss stage will be played in both zones, called the *Regular stage*. There will be `16` teams advance in total, and the others get eliminated from the tournament.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:160]:
```suggestion
2. Following the drafting phase, a Swiss stage will be played in both zones, called the *regular stage*. 16 teams will advance in total, and the others get eliminated from the tournament.
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+3. Following the Regular stage, the Elimination stage will be played, called *Playoffs*. Top 8 teams in regular stage will get into winner bracket, other teams will get into loser bracket.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:161]:
```suggestion
3. Following the regular stage, the elimination stage will be played, called *playoffs*. The top 8 teams from the regular stage will advance to a winner bracket, other teams will get into loser bracket.
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+4. All the losing team in loser bracket will be eliminated from the tournament, the last team will be the winner.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:162]:
```suggestion
4. All losing teams in the loser bracket will be eliminated from the tournament and the last team will be the winner.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
partial review

- please don't use code blocks to format random numbers and text. that's not what they're intended for

**Comment by HanHan3z** [comment]
All mentioned errors have been fixed.

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+   - For the first occurrence:
+     - The team will receive a verbal warning from the referee.
+   - On subsequent occurrences:
+     - For bans of opposing players, the captain forfeits the ban.
+     - For map bans, the captain forfeits the ban.
+     - For map picks, a random map will be chosen from the mappool using `!roll X`, where X is the number of maps that were neither picked nor banned, excluding the tiebreaker.
+     - For a ready timer, the referee will issue the `!mp start 10` command, regardless of how many players from each team are present in the lobby, using `!mp kick` on any extra players for each team, starting from the top (i.e. the first valid player combination for each team will be forced to play the pick). The results for such games are to be taken as is.
+   - Repeat offenders may receive further sanctions from the host.
+8. Each team will receive **one** "tactical timeout" of **two minutes**, to be used as extra time to pick or ban a beatmap. *The tactical timeout is optional.*
+   - The tactical timeout may be called on a tiebreaker, provided that both teams' captains agree to it.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:245]: this rule conflicts with rule 6. please remove one of them and adjust accordingly

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+
+### Mappool instructions
+
+1. There will be five separate mappools. *This might change depending on the number of incoming registrations.*
+   - Qualifiers will use a separate mappool.
+   - Regular stage Round 1 and Round 2 use mappool \#1.
+   - Regular stage Round 3 and Round 4 use mappool \#2.
+   - Round of 16 and Quarterfinals use mappool \#3.
+   - Semifinals and Finals use mappool \#4.
+   - This might change.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:255]:
```suggestion
   - Qualifiers will use a separate mappool.
   - Regular stage round 1 and round 2 use mappool #1.
   - Regular stage round 3 and round 4 use mappool #2.
   - Round of 16 and Quarterfinals use mappool #3.
   - Semifinals and Finals use mappool #4.
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 6]
```diff
@@ -0,0 +1,283 @@
...
+2. Each mappool consists of a fixed amount of maps each stage which will all be played under [FreeMod](/wiki/Gameplay/Game_modifier#freemod) conditions. This means that there is a unique FreeMod bracket.
+3. The mappool sizes are as follows:
+   - Qualifiers: 8 beatmaps   
+   - Mappool \#1 & \#2: 15 beatmaps
+   - Mappool \#3 & \#4: 17 beatmaps.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:260]:
```suggestion
   - Mappool #1 & #2: 15 beatmaps
   - Mappool #3 & #4: 17 beatmaps.
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 7]
```diff
@@ -0,0 +1,283 @@
...
+4. Each mappool has one tiebreaker, except for the Qualifiers.
+5. Possible mod choices for the FreeMod bracket are `NoFail`, `Hidden`, `Fade In`, `Flashlight`, and `Mirror`.
+6. The tiebreaker will be played under FreeMod conditions.
+
+### Scheduling information
+
+1. Each stage will be held on **a single weekend**.
+2. Matches in Qualifiers will be held during specific time slots proposed by the tournament staff. Each player is free to choose a time slot in which they want to play.
+   - **Players that don't pick a time slot by Wednesday, 11 October 23:59 UTC+8 will be disqualified from the competition.**
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:269]:
```suggestion
   - **Players that do not pick a time slot by Wednesday, 11 October 23:59 UTC+8 will be disqualified from the competition.**
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 8]
```diff
@@ -0,0 +1,283 @@
...
+3. All matches will be held between Friday 18:00 and Sunday 23:59 (UTC+8).
+   - Rescheduling a match to a weekday is only permitted if **both** teams request and agree to it.
+4. Scheduling will be handled by the host. Schedules will be released on the Sunday before the first match of the stage. The host will try to create a schedule that respects all participants' preferences.
+   - Please inform the host before Sunday if you expect a specific time slot to be unavailable in the following week. The host will try to accommodate all wishes, but makes no promises.
+5. **Reschedules will only be considered if both teams agree to a time and communicate it to the tournament staff, before Wednesday 23:59 UTC+8 of the week the match is to take place on.**
+   - Reschedules may only be requested by a team captain.
+   - Matches may not be scheduled to any time beyond **Sunday, 23:59 UTC+8** of the week they are to be played at.
+     - ... Unless its absolutely necessary, **both** teams request and agree to it, and staff avaliable.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:277]: already mentioned in rule 3 above

```suggestion
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 9]
```diff
@@ -0,0 +1,283 @@
...
+5. Possible mod choices for the FreeMod bracket are `NoFail`, `Hidden`, `Fade In`, `Flashlight`, and `Mirror`.
+6. The tiebreaker will be played under FreeMod conditions.
+
+### Scheduling information
+
+1. Each stage will be held on **a single weekend**.
+2. Matches in Qualifiers will be held during specific time slots proposed by the tournament staff. Each player is free to choose a time slot in which they want to play.
+   - **Players that don't pick a time slot by Wednesday, 11 October 23:59 UTC+8 will be disqualified from the competition.**
+3. All matches will be held between Friday 18:00 and Sunday 23:59 (UTC+8).
+   - Rescheduling a match to a weekday is only permitted if **both** teams request and agree to it.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:271]:
```suggestion
   - Rescheduling a match to a weekday is only permitted if absolutely necessary, **both** teams request and agree to it, and staff members are available.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md [hunk 10]
```diff
@@ -0,0 +1,270 @@
...
+7. 资格赛的结果将会用于选择队长和队长选人。
+   - 更高的资格赛出分代表选秀时更高概率被选走，如果选秀结束后没有被选择则代表选手被淘汰。如果资格赛出分过低也可能会被淘汰。
+8. 排名最高的部分选手才可以进入选秀阶段。
+   - 通过犯规或排名操纵来扰乱比赛将导致取消资格，任何此类选手都将在选秀阶段开始之前公布。
+9. 资格赛在同一时段的选手将会共用同一个房间，同时资格赛不允许因为技术问题的重开，技术问题将直接导致本图成绩保留。
+
+### 阶段介绍
+
+1. 资格赛结束之后，通过的选手将会进入*选秀阶段*进行组队，两个赛区分别组队。未被选择的选手将会被淘汰。
+2. 选秀结束后，两个赛区的队伍将会分别进行四轮瑞士轮*预选赛*，以决出总计 `16` 支队伍进入淘汰赛，其他队伍将被淘汰。
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md:158]: code blocks shouldn't be used like this

```suggestion
2. 选秀结束后，两个赛区的队伍将会分别进行四轮瑞士轮*预选赛*，以决出总计 16 支队伍进入淘汰赛，其他队伍将被淘汰。
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md [hunk 11]
```diff
@@ -0,0 +1,270 @@
...
+   - 两个赛区的选人阶段将分开举行，队伍数量由本赛区通过资格赛的选手数量决定。
+2. 在选秀阶段队长将会依次挑选他们的队员。
+3. 选秀开始之前会有 120 秒的准备时间。
+4. 选人开始后，所有队长来回选四轮，第一轮第三轮按正序选择，第二轮第四轮按倒序选择。
+5. 每个队长有 60 秒的时间来从选手池中选择一名选手，如果没选，那么他会被随机分派一名选手。
+6. 每轮选完之后，有 60 秒的准备时间，随后下一轮开始。
+7. 在选秀过程中，队长可以在聊天室里面自由发言，其他所有选手池的玩家将被禁言。
+8. 选秀结束之后，队长可以在告知主办方之后将队长位置让给其他队员。
+9. 队长需要在选秀结束后决定队伍的名称。
+   - 默认名称是`Team {X}`, `{X}`是队长用户名的缩写。
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md:175]:
```suggestion
   - 默认名称是 `Team {X}`, `{X}` 是队长用户名的缩写。
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md [hunk 12]
```diff
@@ -0,0 +1,270 @@
...
+   - 队名最长不得超过 **20** 字符(中文字符算作两个字符)。
+10. 在选秀阶段的不良行为，将被视为通过犯规扰乱比赛。
+
+### 预选赛介绍
+
+1. 预选赛将划分为两个赛区，分别为使用 Discord 的亚洲赛区和使用 QQ 的中国赛区.
+   - 选手可以在报名时自行选择参加的赛区并参与对应赛区的组队，选手必须保证可以使用对应赛区的聊天工具并保持消息畅通。
+2. 常规赛总共包含四轮瑞士轮比赛，这意味着比赛期间场次成绩相同的队伍会优先进行比赛。
+   - 两个赛区将分别进行瑞士轮预选赛的比赛，决出指定数量的队伍进入淘汰赛，每个赛区进入淘汰赛的队伍数量由报名人数决定，两个赛区合计固定为 `16` 支队伍进入淘汰赛。
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md:184]:
```suggestion
   - 两个赛区将分别进行瑞士轮预选赛的比赛，决出指定数量的队伍进入淘汰赛，每个赛区进入淘汰赛的队伍数量由报名人数决定，两个赛区合计固定为 16 支队伍进入淘汰赛。
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md [hunk 13]
```diff
@@ -0,0 +1,270 @@
...
+  - GBC2023
+  - GBC 2023
+  - GBC 2023 Autumn
+---
+
+# GBC 2023 秋季赛
+
+![GBC 2023 Autumn banner](img/GBC-2023-Autumn-Banner.png)
+
+**GBC 2023 秋季赛** (***GB 杯 2023 秋季赛***) 是由 GB 组织主办的 3v3 队长选秀制 osu!mania4k 项目赛事。本届比赛是 GBC 系列赛事的第四届正式比赛。
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md:13]:
```suggestion
**GBC 2023 秋季赛** (***GB 杯 2023 秋季赛***) 是由 GB 组织主办的 3v3 队长选秀制 osu!mania 4K 项目赛事。本届比赛是 GBC 系列赛事的第四届正式比赛。
```

**Comment by HanHan3z** [comment]
All mentioned errors have been fixed again.

**Comment by HanHan3z** [comment]
All mentioned errors have been fixed again.

File: wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md [hunk 14]
```diff
@@ -0,0 +1,267 @@
...
+| *第 5-8 名* | *TBA* |
+| *第 9-16 名* | *TBA* |
+| *第 17-21 名* | *TBA* |
+| *第 22-24 名* | *TBA* |
+
+## 组织
+
+GBC 2023 秋季赛由 GB 成员和多位社区成员举办。
+
+| Position | Member(s) |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md:51]: should be translated

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 15]
```diff
@@ -0,0 +1,278 @@
...
+2. Captains decide their team members during the drafting phase.
+   - Captains are allowed to conduct and coordinate "tryouts", tests to gauge player aptitude, on their own terms. 
+3. There will be a 120-second planning phase before the picking phase starts.
+4. During the picking phase, the pick order consists of 2 phases that repeat twice, such that every captain gets to choose 4 players.
+   - During phase 1 and 3, picks start in *increasing* order, from the 1st to the 24th captain.
+   - During phase 2 and 4, picks start in *decreasing* order, from the 24th to the 1st captain.
+   - *This might change depending on the number of incoming registrations.*
+5. Each captain has 60 seconds to pick one player from the player pool on their turn. 
+   - If the captain does not pick a player within 60 seconds, a random player from the remaining player pool will be assigned to their team.
+6. If the captain picks a player, the pick time still has to elapse before the next captain's turn starts, meaning each phase will take 16 minutes.
```
Comment (RushFTK) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:178]: Untranslated.

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 16]
```diff
@@ -0,0 +1,278 @@
...
+4. All losing teams in the loser bracket will be eliminated from the tournament and the last team will be the winner.
+
+### Drafting phase instructions
+
+1. The host will declare one captain for each team 24 hours before the drafting phase.
+   - Drafting order will also be declared by the host.
+   - Captains are selected by performance and preference.
+   - Non-captains will join a group of players waiting to join a team, herein called a "player pool".
+2. Captains decide their team members during the drafting phase.
+   - Captains are allowed to conduct and coordinate "tryouts", tests to gauge player aptitude, on their own terms.
```
Comment (RushFTK) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md]: Untranslated, but dosen't matter i guess. this is just an reminder for captains.

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 17]
```diff
@@ -0,0 +1,278 @@
...
+   - The team mentioned first in the room name must be the red team, and the team mentioned second in the room name must be the blue team.
+2. Each captain can protect **one beatmap** from the pool. These beatmaps cannot be banned. Then, each captain can ban **one beatmap** from the pool. These beatmaps cannot be picked by any team for the entire duration of the match.
+3. Both captains will alternate picking a beatmap from the mappool.
+4. Each captain must use `!roll` once in `#multiplayer`.
+   - The winner of the `!roll` may choose to pick first or ban first in the match. The other choice will be used by loser of the roll. The team who picks first will also protect first, and the others will ban first.
+5. After each pick, both captains **may** ban **one** of the opposing team members from playing the beatmap in a match, **except during tiebreakers**.
+   - Any player can only be banned like this **once** per match.
+   - Only players who have already played in the match can be banned.
```
Comment (RushFTK) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:227]: Untranslated, is this rules changed?

**Review by RushFTK — CHANGES_REQUESTED** [review]
Quickly reviewed requested by @Walavouchey , semmed fines for me in most cases.

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md (hunk 15 above)
Comment (RushFTK) [reply wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:178]: fixed in 689aebb

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md (hunk 16 above)
Comment (RushFTK) [reply wiki/Tournaments/GBC/GBC_2023_Autumn/en.md]: fixed in 689aebb

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md (hunk 17 above)
Comment (RushFTK) [reply wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:227]: fixed on [3704f72](https://github.com/ppy/osu-wiki/pull/10075/commits/3704f7262ef29ea324bb6e985a23fd11d3930204)

**Review by RushFTK — APPROVED** [review]
All inconsist translation have been fixed now

**Comment by Walavouchey** [comment]
force-pushed to condense ~40 commits

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**